File: pinyougou_parent/pinyougou_manager_web/src/main/webapp/js/service/brandService.js
app.service("brandService",function ($http) {
    this.findAll=function () {
        return $http.get("../brand/findAll.do");
    }

    this.save=function (entity) {
        return $http.post('../brand/save.do',entity);
    }

    this.updat = function (entity) {
        return $http.post('../brand/update.do',entity);
    }

    this.findOne = function (id) {
        return $http.get('../brand/findOne.do?id='+id)
    }

    this.dele = function (lists) {
        return $http.get('../brand/dele.do?ids='+lists);
    }

    this.findAllByMany = function (page,size,selectMany) {
        return $http.post('../brand/findByManyAndPages.do?pageNum='+page +'&size='+size , selectMany)
    }

    this.selectListByBrand=function () {
        return $http.get("../brand/selectOptionList.do");

    }
})

File: pinyougou_parent/pinyougou_sellergoods_interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbBrand;
import entity.PageResult;

import java.util.List;
import java.util.Map;

public interface BrandService {
    List<TbBrand> findAll();

    PageResult findAndPages(Integer pageNum,Integer pageSize);

     void save(TbBrand tbBrand);

     void update(TbBrand tbBrand);

     TbBrand findOne(Long id);

     void dele(Long[] ids);

    PageResult findByManyAndPages(TbBrand tbBrand,Integer pageNum,Integer pageSize);

    List<Map>  selectOptionList();
}
